wire-ios/Wire-iOS Tests/DeviceManagement/ClientTableViewCellTests.swift

```diff
@@ -145,10 +145,18 @@ final class ClientTableViewCellTests: BaseSnapshotTestCase {
 extension ClientTableViewCellModel {
     typealias DeviceDetailsSection = L10n.Localizable.Device.Details.Section
 
+    private static let mockProteusId: String = "abcdefghijklmnop"
+        .uppercased()
+        .splitStringIntoLines(charactersPerLine: 16)
+
+    private static let mockFingerPrint: String = "abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijkl"
+        .uppercased()
+        .splitStringIntoLines(charactersPerLine: 16)
+
     static func mock(title: String = "Lorem ipsum",
                      isProteusVerified: Bool = false,
-                     mlsThumbprint: String = .mockFingerPrint,
```

Comment:
**question:**  any reasons why we removed the common mocks and using explicit mocks here ?

Reply:
Yes, the reason is that it is not necessary to add a dependency to a definition in another file. Once you want to move files around (e.g. move class to another module) this becomes very complex and is the main reason why I could not easily split unit tests and snapshot tests in my tries.